pkg/apis/minio.min.io/v2/types.go

```diff
@@ -284,6 +269,33 @@ type TenantSpec struct {
 	// Specify custom labels and annotations to append to the MinIO service and/or Console service.
 	// +optional
 	ServiceMetadata *ServiceMetadata `json:"serviceMetadata,omitempty"`
+	// *Optional* +
+	//
+	// An array of https://kubernetes.io/docs/concepts/configuration/secret/[Kubernetes opaque secrets] to use for generating MinIO users during tenant provisioning. +
+	//
+	// Each element in the array is an object consisting of a key-value pair `name: <string>`, where the `<string>` references an opaque Kubernetes secret. +
+	//
+	// Each referenced Kubernetes secret must include the following fields: +
+	//
+	// * `CONSOLE_ACCESS_KEY` - The "Username" for the MinIO user +
+	//
+	// * `CONSOLE_SECRET_KEY` - The "Password" for the MinIO user +
+	//
+	// The Operator creates each user with the `consoleAdmin` policy by default. You can change the assigned policy after the Tenant starts. +
+	// +optional
+	Users []*corev1.LocalObjectReference `json:"users,omitempty"`
+	// *Optional* +
+	//
+	// Enable JSON, Anonymous logging for MinIO tenants.
+	// +optional
+	Logging *Logging `json:"logging,omitempty"`
```

Comment:
We need to add this on the `types.go` for `v1` as well

Reply:
Also in `pkg/apis/minio.min.io/v1/conversion.go` you should add support to copy this new field from v1 to v2 and viceversa

Reply:
why v1? @dvaldivia